packages/react/src/components/Card/index.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+const ELEVATIONS = [0, 1, 2];
+
+const Card = ({ className, elevation, isSelected, ...props }) => (
+  <div
+    className={cx("Card", className, {
+      "is-selected": isSelected,
+      [`is-elevated--${elevation}`]: elevation,
```

Comment:
```suggestion
      [`is-elevated--${elevation}`]: ELEVATIONS.includes(elevation),
```
?

Reply:
@pppemo won't that return true instead of the number?

Reply:
ah sorry I see what's happening, you actually want it to return "true"

Reply:
Yeah, you want true. If it's a number, JS just casts it to boolean. But generally the value here is expected to be `boolean`.
And I thought that would be another guard (next to `propTypes`) that would cover a scenario, when user invokes `elevation={66609}` for example :)

Reply:
Got you! Thanks for the insight.

Reply:
I was about to say that `0` is casts to `false`, but +1 for the extra scenario 🙂